### DISCUSSION TITLE
SelectElement: Trigger updateItems when select is reset if items is a function

### DISCUSSION THREAD

**Discussion by nclemeur** [vueform/vueform#187] [Ideas]
I would like to propose that the updateItems is also triggered on a reset event if the items is a function. I think it make sense to also include functions as they are not really different from an endpoint configuration: Check the code [here](https://github.com/vueform/vueform/blob/f803c9e87ebb3c82693f4abdaa63140f5ff8a678/src/composables/elements/useData.js#L276)

**Comment by adamberecz** [comment]
Assigned for the Ideas category. Thanks for the suggestion.
